cmd/support-logging/res/docker/configuration.toml

```diff
@@ -23,7 +23,7 @@ File = '/edgex/logs/edgex-support-logging.log'
 
 [Databases]
   [Databases.Primary]
-  Host = 'edgex-mongo'
+  Host = 'edgex-redis'
   Name = 'logging'
   Password = 'password'
   Port = 27017
```

Comment:
I think we may be able to get rid of the `Databases` section since we are now using `Persistence = 'file'`.

Relevant code: https://github.com/edgexfoundry/edgex-go/blob/e55283042323a2981925e440e68f7dc71be36291/internal/support/logging/init.go#L58-L65

Reply:
I don't know about this.  

Mongo is still an option and we need to ensure that's documented somewhere.  This is a follow-on to a similar conversation in a separate issue within the last 24 hours.  I escalated that issue to the Architecture WG for discussion.

There's also https://github.com/edgexfoundry/edgex-go/issues/1903 outstanding and the open question of whether or not to leverage Redis for logging.

Reply:
We may need a better documentation strategy for configuration. I agree that it needs to be somewhere. I just don't like the idea of having configuration properties present that are not actually doing anything. Could we maybe comment out the section and put a note what the section is used for and when the service takes it into consideration?

Reply:
I agree -- I don't like the idea of having configuration properties present that serve no configuration purpose.  Even the idea of commenting out the section feels icky to me.

I'm hesitant to merge this until (1.) we have a clear picture of the path forward with respect to support-logging persisting logging entries in Redis, and (2.) we have a clear picture of how we should consistently document alternative configuration settings (especially for those situations where an external library determines the specifics).

I have requested (1.) be added to the next Core WG meeting.  I've separately asked for (2.) to be added (as it relates to a separate issue that came up in the last 24 hours) to the next Architect's WG agenda.